### PR TITLE
infinite scroll on the invites page

### PR DESCRIPTION
**Pull request by gmethvin**

### CODE REVIEW

File: kifi-backend/modules/common/public/site/js/main.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-var xhrDomain = 'https://api.kifi.com';
+var xhrDomain = 'http://dev.ezkeep.com:9000';
```
Comment (2is10) [inline kifi-backend/modules/common/public/site/js/main.js:1]: hope this was reverted soon after

Comment (gmethvin) [inline kifi-backend/modules/common/public/site/js/main.js:1]: yes it was

Comment (gmethvin) [inline kifi-backend/modules/common/public/site/js/main.js:1]: need to have a better way of dealing with this

Comment (2is10) [inline kifi-backend/modules/common/public/site/js/main.js:1]: You could put an override in localStorage.

(I just always develop with production data.)

File: kifi-backend/modules/shoebox/app/com/keepit/controllers/website/UserController.scala [hunk 2]
```diff
@@ -287,14 +288,22 @@ class UserController @Inject() (
...
-    val connections = db.readOnly { implicit s =>
+    val unfilteredConnections = db.readOnly { implicit s =>
       socialUserRepo.getByUser(request.userId)
         .flatMap(getFilteredConnections)
         .map(getWithInviteStatus)
         .sortBy { case (sui, status) => (status, -searchScore(sui), normalize(sui.fullName)) }
-        .take(limit)
     }
 
+    println(unfilteredConnections)
```
Comment (2is10) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/UserController.scala:298]: `println`?

**Comment by 2is10** [comment]
Infinite scroll for search results too? Nice.

File: kifi-backend/modules/shoebox/app/com/keepit/controllers/website/UserController.scala (hunk 2 above)
Comment (gmethvin) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/UserController.scala:298]: oops; removed

**Pull request merged**